src/herder/TxQueueLimiter.cpp

```diff
@@ -5,16 +5,22 @@
 #include "herder/TxQueueLimiter.h"
 #include "crypto/ShortHash.h"
 #include "util/GlobalChecks.h"
+#include "util/numeric.h"
 
 namespace stellar
 {
 
 TxQueueLimiter::TxQueueLimiter(int multiplier, int percentThresholdRecovery,
-                               LedgerManager& lm)
+                               int feeMultiplier, LedgerManager& lm)
     : mPoolLedgerMultiplier(multiplier)
     , mThresholdFreePercentUnstick(percentThresholdRecovery)
+    , mBaseFeeBidMultiplier(feeMultiplier)
     , mLedgerManager(lm)
 {
+    if (mBaseFeeBidMultiplier < 1)
```

Comment:
Either the condition is wrong or the exception is wrong. Do you want to accept 1? If so, the error message should probably say "Multiplier must be at least 1"

You might want to check the same for `mPoolLedgerMultiplier`.